src/examples/full.yml

```diff
@@ -0,0 +1,96 @@
+description: A full example of running integration tests and publishing a new orb version.
+
+usage:
+  version: 2.1
+
+  orbs:
+    orb-tools: circleci/orb-tools@10
+    # add your orb below, to be used in integration tests (note: a
+    # @dev:alpha release must exist; if none exists, you'll need to
+    # publish manually once before this worklow can succeed)
+    your-orb: your-namespace/your-orb@<<pipeline.parameters.dev-orb-version>>
+
+  orb_prep_jobs: &orb_prep_jobs
+    [
+      orb-tools/lint,
+      orb-tools/shellcheck,
+      orb-tools/pack
+    ]
+
+  # Pipeline parameters
+  parameters:
+    # These pipeline parameters are required by the "trigger-integration-tests-workflow"
+    # job, by default.
+    run-integration-tests:
+      type: boolean
+      default: false
+    dev-orb-version:
+      type: string
+      default: "dev:alpha"
+
+  jobs:
+    integration-tests-for-your-orb:
+      executor: orb-tools/ubuntu
+      steps:
+        - checkout
+      # Test your orb e.g.
+      # - your-orb/your-orb-command
+
+  workflows:
+    lint_pack-validate_publish-dev:
```

Comment:
...should this be update to match the config?

Reply:
Good catch :)